src/App.tsx

```diff
@@ -15,71 +18,95 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+type State = {
+  selectedGood: string,
+};
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+export class App extends Component<{}, State> {
+  state = {
+    selectedGood: '',
```

Comment:
Jam should be selected by default
![image](https://github.com/mate-academy/react_goods-selector-js/assets/96409563/fb12227e-9f6f-430b-80f8-c7d93dc530ec)